<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordPage</class>
 <widget class="QWidget" name="RecordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="CustomImageLabel" name="buttonNormal">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>160</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNormalText">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>410</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonEvent">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>160</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="CustomImageLabel" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>839</x>
     <y>160</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEventText">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>410</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCloseText">
   <property name="geometry">
    <rect>
     <x>859</x>
     <y>410</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <zorder>labelCloseText</zorder>
  <zorder>labelNormalText</zorder>
  <zorder>labelEventText</zorder>
  <zorder>buttonNormal</zorder>
  <zorder>buttonEvent</zorder>
  <zorder>buttonClose</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomImageLabel</class>
   <extends>QLabel</extends>
   <header>ui/customwidget/customimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
